data/json/recipes/recipe_food.json

```diff
@@ -10056,5 +10056,20 @@
     "autolearn": true,
     "tools": [ [ [ "food_processor", 20 ] ] ],
     "components": [ [ [ "bone_tainted", 1 ] ] ]
+  },{
+    "result": "cattlefodder",
+    "type": "recipe",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 2,
+    "skills_required": [ "survival", 1 ],
+    "time": 10000,
+    "autolearn": true,
+    "components": [
+      [ [ "oats", 1 ], [ "buckwheat", 1 ], [ "wheat", 1 ], [ "potato_raw", 1 ], [ "barley", 1 ], [ "raw_beans", 1 ], [ "cabbage", 1 ], [ "veggy_wild", 1 ], [ "veggy", 1 ] ],
+      [ [ "oats", 1 ], [ "buckwheat", 1 ], [ "wheat", 1 ], [ "potato_raw", 1 ], [ "barley", 1 ], [ "raw_beans", 1 ], [ "cabbage", 1 ], [ "veggy_wild", 1 ], [ "veggy", 1 ] ],
+      [ [ "oats", 1 ], [ "buckwheat", 1 ], [ "wheat", 1 ], [ "potato_raw", 1 ], [ "barley", 1 ], [ "raw_beans", 1 ], [ "cabbage", 1 ], [ "veggy_wild", 1 ], [ "veggy", 1 ] ]
```

Comment:
This will not work - the crafting code doesn't understand multiple components of the same type.